sup3r/preprocessing/data_handling.py

```diff
@@ -1106,7 +1106,7 @@ def preflight(self):
         msg = (f'sample_shape[2] ({self.sample_shape[2]}) cannot be larger '
                'than the number of time steps in the raw data '
                f'({len(self.raw_time_index)}).')
-        if len(self.raw_time_index) >= self.sample_shape[2]:
+        if len(self.raw_time_index) < self.sample_shape[2]:
```

Comment:
good catch, fixed this in my current dev branch also.

Reply:
it was driving me insane haha